packages/mnemonic/src/mnemonic.test.ts

```diff
@@ -96,4 +96,17 @@ describe('Mnemonic', () => {
     const phrase = Mnemonic.generate(32, randomBytes(16));
     expect(phrase.split(' ').length).toBe(24);
   });
+
+  test("Validate a incomplete Mnemonic phrase" , () => {
+    const mnemonicphrase = "Hello my name is Fuel"
+    const validation = Mnemonic.isValidMnemonic(mnemonicphrase);
+    expect(validation == false);
+  })
+
+  test("Validate a complete Mnemonic phrase" , () => {
+    const mnemonicphrase = "aware fringe excess tank asset quick suffer second cloth deliver file above"
```

Comment:
Can you add tests for 12, 14, 16, 18, and 24 seed phrases?

Reply:
Does >12 are valid in Fuel?

Reply:
Yes, we accept any size of mnemonic. You can check on the method from Mnemonic.https://github.com/FuelLabs/fuels-ts/blob/d0eb1c732f63842b8d4801456054ec3b9ccdd020/packages/mnemonic/src/mnemonic.ts#L39

Reply:
That's cool!! I added those and used that function